src/utilities/dailyUsage.js

```diff
@@ -108,5 +108,5 @@ export const programDailyUsage = (item, period) => {
  */
 export const customerRequisitionProgramDailyUsage = requisitionItem => {
   const { outgoingStock, daysOutOfStock, numberOfDaysInPeriod } = requisitionItem;
-  return Math.max(outgoingStock / (numberOfDaysInPeriod ?? 0 - daysOutOfStock), 0);
+  return Math.max(outgoingStock / ((numberOfDaysInPeriod ?? 0) - daysOutOfStock), 0);
```

Comment:
Maybe a check for the case where `numberOfDaysInPeriod` is 0 and `daysOutOfStock` is too - returning `0` in that case makes more sense than infinity..maybe?

Reply:
Ah of course, Doh! I added more slightly (too?/more?) defensive code .. 
feeb4a4